bench/src/main/scala/Benchmarks.scala

```diff
@@ -37,7 +37,7 @@ object Bench {
     val libs = System.getProperty("BENCH_CLASS_PATH")
 
     val opts = new OptionsBuilder()
-               .jvmArgsPrepend("-Xbootclasspath/a:" + libs + ":", "-Xms2G", "-Xmx2G")
+               .jvmArgsPrepend("-Xbootclasspath/a:" + libs + ":", "-Xms2G", "-Xmx2G", "-XX:MaxInlineLevel=35")
```

Comment:
This is fine for testing but we shouldn't merge this since it doesn't reflect what 99% of people using dotty through sbt will get.

Reply:
I there a way to add this to sbt?